mirrord/layer/src/file/filter.rs

```diff
@@ -112,9 +104,9 @@ fn generate_remote_ro_set() -> RegexSet {
     let patterns = [
         // AWS cli cache
         // "file not exist" for identity caches (AWS)
-        r".aws/cli/cache/.+\.json$",
-        r".aws/credentials$",
-        r".aws/config$",
+        r"\.aws/cli/cache/.+\.json$",
```

Comment:
the aws here was actually a patch to return not found by making it doing it remote, so you can remove it :)